crates/oci-distribution/src/client.rs

```diff
@@ -261,8 +260,6 @@ impl ClientProtocol {
 #[derive(serde::Deserialize, Default)]
 struct RegistryToken {
     access_token: String,
-    expires_in: Option<u32>,
```

Comment:
I know these are unused now, but I think we will need them later. That correct @radu-matei?

Reply:
I originally added `[allow(dead_code)]` because I figured we would need them eventually but then I just removed them because adding them back should be easy.

Reply:
Ok, I think that is fine then